e2e_deployment_files/stress_deployment.template.json

```diff
@@ -170,16 +170,16 @@
                 "value": "<Analyzer.ConsumerGroupId>"
               }, 
               "LogAnalyticEnabled": {
-                "value": "<Analyzer.LogAnalyticEnabled>"
+                "value": "<Analyzer.LogAnalyticsEnabled>"
               },
               "LogAnalyticWorkspaceId": {
-                "value": "<Analyzer.LogAnalyticWorkspaceId>"
+                "value": "<LogAnalyticsWorkspaceId>"
               },
               "LogAnalyticSharedKey": {
```

Comment:
I think you missed to update this environment variable key

Reply:
Let me retest on agents and I will let you know when done. Initial test didn't throw exception because inconsistent keys disabled log analytics (which led to no post logic running). I was looking for exceptions, but should also have looked for the data in log analytics itself. My bad